### ISSUE TITLE
"Incompatible geometry" error with readOGR()

### ISSUE THREAD

**Issue by yutannihilation** [yutannihilation/kokudosuuchi#3]
```r
getKSJData('http://nlftp.mlit.go.jp/ksj/gml/data/P12/P12-14/P12-14_36_GML.zip')
trying URL 'http://nlftp.mlit.go.jp/ksj/gml/data/P12/P12-14/P12-14_36_GML.zip'
Content type 'application/zip' length 47274 bytes (46 KB)
downloaded 46 KB

OGR data source with driver: ESRI Shapefile 
Source: "C:/Users/User1/AppData/Local/Temp/RtmpgjkLRH/e2d2fce8c1166e635c13acd5ceb3adfa/P12-14_36_GML", layer: "P12a-14_36一般公開"
with 245 features
It has 7 fields
 Show Traceback
 
 Rerun with Debug
 Error in rgdal::readOGR(data_dir, layer, encoding = encoding) : 
  Incompatible geometry: 4 
```

**Comment by yutannihilation** [comment]
This seems to be a rgdal's bug. Geometry 4 means MultiPoint, which should be treated in the same way as Point.
- http://www.gdal.org/ogr__core_8h.html#enum-members
- https://github.com/cran/rgdal/blob/9337eddc5d62d1eff6cbac98a488a9b3cdbacdf8/R/ogr_sp.R#L224-L225

**Comment by yutannihilation** [comment]
Try maptools.

**Comment by yutannihilation** [comment]
This still won't work.

``` r
getKSJData('http://nlftp.mlit.go.jp/ksj/gml/data/P12/P12-14/P12-14_05_GML.zip')
```

**Comment by yutannihilation** [comment]
Use sf.

**Comment by yutannihilation** [comment]
fixed by #4
